### PR TITLE
plots: smoke test for happy case

### PR DESCRIPTION
**Pull request by wwwillchen**
A simple smoke test for plots. Will add more over time.

Closes part of #1980

### CODE REVIEW

File: plots/test/smoke.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+if [ -d "$out_path" ]; then
+  mv $out_path $out_backup_path
+  echo "Moved existing plots/out folder to plots/$out_backup_folder"
+fi
+
+node $measure_script --site https://google.com/ -n 2 --disable-network-throttling --disable-cpu-throttling
+node $analyze_script --ci
+node $clean_script
+
+echo "Finished plots smoke test without errors"
```
Comment (patrickhulce) [inline plots/test/smoke.sh]: dontcha need some `set -e` goodness to make this fail?

Comment (wwwillchen) [inline plots/test/smoke.sh]: yup.

File: plots/analyze.js [hunk 2]
```diff
@@ -45,6 +48,9 @@ function main() {
     GENERATED_RESULTS_PATH,
     `var generatedResults = ${JSON.stringify(generatedResults)}`
   );
+  if (args.ci) {
```
Comment (paulirish) [inline plots/analyze.js]: you can do 

```js
if (process.env.CI)
```

and avoid adding a custom flag

File: plots/test/smoke.sh [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+echo "Starting plots smoke test"
+
+if [ -d "$out_path" ]; then
+  mv $out_path $out_backup_path
+  echo "Moved existing plots/out folder to plots/$out_backup_folder"
+fi
+
+node $measure_script --site https://google.com/ -n 2 --disable-network-throttling --disable-cpu-throttling
+node $analyze_script --ci
```
Comment (paulirish) [inline plots/test/smoke.sh]: do you want to assert anything about the analyze results?

File: plots/analyze.js (hunk 2 above)
Comment (wwwillchen) [inline plots/analyze.js]: Done

File: plots/test/smoke.sh (hunk 3 above)
Comment (wwwillchen) [inline plots/test/smoke.sh]: I'll do this is in a follow-up PR - I think many of the potential issues (e.g. lighthouse changes that break plots) would be caught without assertions. 

I'm thinking about processing the output and replace non-deterministic values (e.g. timing values) and then diff it with a golden file.

**Comment by paulirish** [comment]
Only +30s to travis times. Not bad. :)

**Pull request merged**